Runtimes/Core/SwiftOnoneSupport/CMakeLists.txt

```diff
@@ -42,3 +42,4 @@ install_swift_interface(swiftSwiftOnoneSupport)
 
 # Configure plist creation for Darwin platforms.
 generate_plist("${CMAKE_PROJECT_NAME}" "${CMAKE_PROJECT_VERSION}" swiftSwiftOnoneSupport)
+embed_manifest(swiftSwiftOnoneSupport)
```

Comment:
Sorry for missing this in the first place

Reply:
No worries! All part of the bring up :)